### DISCUSSION TITLE
auto-save buffer when idle

### DISCUSSION THREAD

**Discussion by donneyluck** [abougouffa/minemacs#137] [Ideas]
This package can save the buffer when idle. The effect is good. You can try it.
https://github.com/manateelazycat/auto-save

**Comment by abougouffa** [comment]
Hi @donneyluck ,

Thank you for the package, it provides exactly what I was looking for! I've used `super-save` for a while but the thing that bite me up was the auto cleanup of white spaces on the current line. Some times when I'm at the middle of writing some thing and I switch to another window, it would trim the space from the current line. The `auto-save` package have the right hack for this with it's 

```elisp
(defcustom auto-save-delete-trailing-whitespace nil
  "Delete trailing whitespace when save if this option is non-nil.
Note, this option is non-nil, will delete all training whitespace execpet current line,
avoid delete current indent space when you programming."
  :type 'boolean
  :group 'auto-save)
```

I've added support for `auto-save` in the `me-editor` module. I will push it soon. Thank you again!

**Comment by abougouffa** [comment]
Done in ef19196cf27b261a305be9f4f60ae8aa9f6b2a11, thank you again!

**Comment by abougouffa** [comment]
Just to know @donneyluck ,

I've switched back to `super-save` which is more flexible. Although, I've ported the nice features from `auto-save` that allows deleting whitespaces and running silently. I'm using my `super-save` fork until my PR in `super-save` gets merged.

**Comment by donneyluck** [comment]
Okay, I understand everything. Thank you again for your configuration.
